### ISSUE TITLE
Tel Aviv AWS Region not supported

### ISSUE THREAD

**Issue by chadish7** [aws-ia/taskcat#806]
**The bug**
When deploying a stack with `taskcat` to the `il-central-1` region the following error is produced:
```
[ERROR  ] : ValidationError 'il-central-1' does not match '^(ap|eu|us|sa|ca|cn|af|me|us-gov)-(central|south|north|east|west|southeast|southwest|northeast|northwest)-[0-9]$'
```

**To Reproduce**
1. Add `il-central-1` to the `regions` section in the`.taskcat.yml` file
2. Deploy the stack with `taskcat test run`

**Questions**

1. Are you testing a *QuickStart* or *Custom template*? No
3. Attach or link a copy of the template if possible (**remove any sensitive info**) N/A
4. Provide the parameters that you passed. (**remove any sensitive info**) N/A
5. How did you install taskcat? pip3
6. Are you using a *profile*, *an instance role* or  *access keys* to run taskcat? profile
7. Is your AWS environment configured via `aws configure`? Yes

**Version**
 - Taskcat Version 0.9.39
 - Python 3.9.17

**Comment by chadish7** [comment]
I see there is [a commit from last week](https://github.com/aws-ia/taskcat/commit/47a43e90708d4afa7295a5bc19125899c64b7da8#diff-ccc66a243e8d964c2cc83304977b07bfe97773f1591bec617f5bcba7319e33c6) that adds support for `il-central-1` but it's not in the release build yet.

**Comment by chadish7** [comment]
Resolved with version `0.9.40`.
